index.js

```diff
@@ -19,11 +19,14 @@ module.exports = () => {
 		};
 	}).catch(() => {
 		return pify(getos).then(res => {
+			if (!res) {
+				res = { };
```

Comment:
@kevva It still can be a "valid" response from `getos`, can't it? They don't support everything. I am trying to get rid of having to wrap my calls to `linux+distro` try-catch. But I guess I should do it anyway, right?